webapp/components/searchable_user_list.jsx

```diff
@@ -30,64 +39,112 @@ export default class SearchableUserList extends React.Component {
         }
     }
 
+    componentWillUnmount() {
+        clearTimeout(this.nextTimeoutId);
+    }
+
     nextPage(e) {
         e.preventDefault();
         this.props.nextPage(this.state.page + 1);
-        this.setState({page: this.state.page + 1});
+        this.setState({page: this.state.page + 1, nextDisabled: true});
```

Comment:
this.props.nextPage could take some time and be async. Maybe set state before calling?